### DISCUSSION TITLE
Any cli or operator based approach to onboard application or namespace instead of UI ?

### DISCUSSION THREAD

**Discussion by rajpratik71** [odigos-io/odigos#1391] [Q&A]
As far I know till now, Below frontend workflow is the approach to onboard a application and namespace 

![image](https://github.com/user-attachments/assets/ae97009c-1f84-479a-a008-348bc4f5a0eb)

I am looking for cli or operator based approach to onboard application or namespace instead of UI , which will help in case of cli based install or automations.

**Comment by edeNFed** [comment]
Hi @Rajpratik71 
You can add additional applications by labeling the Deployment/StatefulSet/DaemonSet object with `odigos-instrumentation=enabled`.
If you want to add all the applications in a specific namespace you can simply put the same label on the namespace instead on every individual application. 
Let me know if that's helps

**Reply by rajpratik71** [reply]
I added at namespace level.

But i am still getting on-boarding page ?

Do i have to wait for some time or it is compulsory to do when using UI ?

**Reply by edeNFed** [reply]
Did you also configured a destination?

**Reply by rajpratik71** [reply]
no not yet. 

Also for same do we have a "cli or operator based approach" for configuring destination ?

**Reply by BenElferink** [reply]
Hey @Rajpratik71 👋 

Yes you can add [supported destinations](https://docs.odigos.io/backends-overview) via the CLI.

See [Jaeger](https://docs.odigos.io/backends/jaeger) for example.
